### DISCUSSION TITLE
Won't start at login

### DISCUSSION THREAD

**Discussion by iosdevben** [MonitorControl/MonitorControl#1655] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Since updating to MacOS Sequoia 15.0.1, Monitor Control does not start at login, despite that feature being enabled.

### Steps to reproduce

Enabled Start at login and reboot the computer.

=> Monitor Control is not running.

### Expected behavior

Monitor Control should be running after rebooting and logging in again.

### Anything else?

![image](https://github.com/user-attachments/assets/d01053fe-abed-4340-b0ae-4c7bfa379164)

### Environment Information (please complete the following information)

```markdown
- macOS version: 15.0.1
- Mac model: 2021 16" MBP M1 Pro 32GB
- MonitorControl version: 4.2.0
- Monitor(s): Dell S2722DC
- Apple Silicon/M1 (yes or no): yes
```

**Comment by wongcken** [comment]
I had the same issue after the update just now. Also, if I manually started up Monitor Control, it can change the brightness of the external monitor by dragging the slider. However, if I use keyboard shortcut or drag the slider over the macbook display, the app crashes.

**Comment by waydabber** [comment]
Make sure you guys are using the latest app version (4.3.3). You might need to re-add the app to Login Items and set up Accessibility Permissions again.

**Reply by iosdevben** [reply]
It reports that I am on the latest version, although that's 4.2.0.

I'll try deleting and reinstalling.

![image](https://github.com/user-attachments/assets/9eacd0cd-c60a-4547-aeeb-25052e5d9832)

**Comment by wongcken** [comment]
Right, I just reinstalled the latest version, it works out fine. Thanks a lot

**Comment by k-matti** [comment]
I still have this issue with the newest version

**Reply by waydabber** [reply]
If the app generally works but does not auto-start, then simply add it to the Login Items manually in System Settings/General.

**Reply by k-matti** [reply]
For some reason, adding app to login items manually doesn’t help. I need to run it manually after every Mac restart.

**Reply by waydabber** [reply]
I see. I am not sure what could be the reason though. :(
